README.md

```diff
@@ -1,4 +1,4 @@
-# rebazer
+# <img src="https://assets.retest.org/retest/ci/logos/retest-wordmark-screen.svg" width="300"/>rebazer
```

Comment:
Do we want to wrap this inside an `<a>` like here? https://github.com/retest/recheck-web/commit/f95aae4646c123bf5ebe6d672fcf3aa5cd7786f3#diff-04c6e90faac2675aa89e2176d2eec7d8R1